integration_tests/support/sharedTests.ts

```diff
@@ -119,6 +140,29 @@ const sharedTests = {
       additionalInformationPage.shouldContainSubmittedText()
     },
 
+    showsEmptyOffenceHistoryPage: (role: ApplicationRole): void => {
+      sharedTests.referrals.beforeEach(role)
+      cy.task('stubOffenderBooking', {
+        offenceHistory: [],
+        offenderNo: prisoner.prisonerNumber,
+      })
+
+      const path = pathsByRole(role).show.offenceHistory({ referralId: referral.id })
+      cy.visit(path)
+
+      const offenceHistoryPage = Page.verifyOnPage(OffenceHistoryPage, {
+        course,
+        person,
+      })
+      offenceHistoryPage.shouldHavePersonDetails(person)
+      offenceHistoryPage.shouldContainNavigation(path)
+      offenceHistoryPage.shouldContainBackLink('#')
+      offenceHistoryPage.shouldContainSubmissionSummaryList(referral)
```

Comment:
Looks like `programmeHistoryPage.shouldContainCourseOfferingSummaryList(coursePresenter, organisation.name)` is missing here and below

Reply:
🦅 👀